api/commands/build.ts

```diff
@@ -55,23 +56,36 @@ export class Builder {
       this.compiledGraph.operations.map(o => this.buildOperation(o)),
       this.compiledGraph.assertions.map(a => this.buildAssertion(a))
     );
-    const allActionNames = allActions.map(n => n.name);
+
     const actionNameMap: { [name: string]: dataform.IExecutionAction } = {};
+
     allActions.forEach(action => (actionNameMap[action.name] = action));
 
+    // Include only tagged actions
+    const taggedActions =
+      this.runConfig.tags && this.runConfig.tags.length > 0
+        ? allActions.filter(action => this.runConfig.tags.some(t => action.tags.includes(t)))
```

Comment:
unless i'm misreading this, i *believe* this logic is equivalent to filtering to only the actions that are in --actions *AND* in --tags. i think that AND should probably be an OR, i.e. if somebody specifies an action in --actions but which does not have a tag in --tags, we should still run it.

Reply:
Yes, you're right. That _AND_ should become an _OR_ because it would not make sense otherwise that the user specified that action using the --action option.